apps/web/vibes/soul/examples/primitives/products-carousel/electric.tsx

```diff
@@ -0,0 +1,185 @@
+import { ProductsCarousel } from '@/vibes/soul/primitives/products-carousel';
+
+export default function Preview() {
+  const products = new Promise<Product[]>((resolve) => {
+    setTimeout(() => resolve(defaultProducts), 1000);
+  });
+
+  return (
+    <div>
+      <section className="overflow-hidden @container">
+        <div className="mx-auto w-full max-w-screen-2xl px-4 py-10 @xl:px-6 @xl:py-14 @4xl:px-8 @4xl:py-20">
+          <ProductsCarousel products={products} />
+        </div>
+      </section>
+      <section className="overflow-hidden bg-foreground @container">
+        <div className="mx-auto w-full max-w-screen-2xl px-4 py-10 @xl:px-6 @xl:py-14 @4xl:px-8 @4xl:py-20">
+          <ProductsCarousel colorScheme="dark" products={products} showScrollbar={false} />
+        </div>
+      </section>
+    </div>
+  );
+}
+
+interface Product {
+  id: string;
+  title: string;
+  subtitle: string;
+  badge?: string;
+  price: string;
+  image: {
+    src: string;
+    alt: string;
+  };
+  href: string;
+  rating: number;
+}
```

Comment:
Don't we export this type from `products-carousel`?

Reply:
Great catch. Removed these duplicate types and used the ones that come from the primitive component